src/components/_checkbox.scss

```
@@ -6,144 +6,144 @@ category: Components
 ---
 
 オプションのオン/オフ切り替えなどをする際に使用します。
-オプションから選択できる項目が２つ以上の場合には(ただし任意選択を除く)、Checkboxの代わりにRadioを使用しましょう。
-
-```example
-<label class="ncgr-checkbox">
-  <input type="checkbox" name="hoge">
-  <span>
-    <i class="fas fa-check"></i>
-    甲賀流
-  </span>
-</label>
-
-<label class="ncgr-checkbox">
-  <input type="checkbox" name="hoge">
-  <span>
-    <i class="fas fa-check"></i>
-    伊賀流
-  </span>
-</label>
+List Viewのleadingやtrailingや、Icon Buttonなどの中に入れて使用することができます。
 
-<label class="ncgr-checkbox">
-  <input type="checkbox" name="hoge">
-  <span>
-    <i class="fas fa-check"></i>
-    その他
-  </span>
-</label>
-```
-
-縦に並べる場合には、以下のようにマークアップします。
+オプションから選択できる項目が２つ以上の場合には(ただし任意選択を除く)、Checkboxの代わりにRadioを使用しましょう。
 
 ```example
-<label class="ncgr-checkbox ncgr-checkbox--vertical">
-  <input type="checkbox" name="hoge">
-  <span>
-    <i class="fas fa-check"></i>
-    甲賀流
-  </span>
-</label>
-
-<label class="ncgr-checkbox ncgr-checkbox--vertical">
-  <input type="checkbox" name="hoge">
-  <span>
-    <i class="fas fa-check"></i>
-    伊賀流
-  </span>
-</label>
-
-<label class="ncgr-checkbox ncgr-checkbox--vertical">
-  <input type="checkbox" name="hoge">
-  <span>
-    <i class="fas fa-check"></i>
-    その他
-  </span>
-</label>
+<div class="ncgr-list-group__subheader">
+  Icon Button
+</div>
+<div class="ncgr-row">
+  <div class="ncgr-col -col-6 -col-s-4">
+      <label class="ncgr-icon-button -circle" for="koga">
+        <div class="ncgr-checkbox">
+          <input type="checkbox" id="koga">
+          <div class="ncgr-checkbox__background">
+            <i class="fas fa-check"></i>
+          </div>
+        </div>
+      </label>
+      甲賀流
+  </div>
+  <div class="ncgr-col -col-6 -col-s-4">
+      <label class="ncgr-icon-button -circle" for="iga">
+        <div class="ncgr-checkbox">
+          <input type="checkbox" id="iga">
+          <div class="ncgr-checkbox__background">
+            <i class="fas fa-check"></i>
+          </div>
+        </div>
+      </label>
+      伊賀流
+  </div>
+  <div class="ncgr-col -col-6 -col-s-4">
+      <label class="ncgr-icon-button -circle" for="others">
+        <div class="ncgr-checkbox">
+          <input type="checkbox" id="others">
+          <div class="ncgr-checkbox__background">
+            <i class="fas fa-check"></i>
+          </div>
+        </div>
+      </label>
+      その他
+  </div>
+</div>
+<hr class="ncgr-list-divider">
+<div class="ncgr-list-group__subheader">
+  ListView
+</div>
+<div class="ncgr-list-view -dense">
```

`-dense` がデフォルトっぽく見えるのでなくてよさそう

```suggestion
<div class="ncgr-list-view">
```